src/ec/packages/ec-component-search-form/search-form.html.twig

```diff
@@ -0,0 +1,37 @@
+{#
+  Parameters:
+    - "textInput" (associative array) default: A predefined structure for EC Text Input
+    - "button" (associative array) default: A predefined structure for EC Button
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{% set _css_class = ['ecl-search-form'] %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+<form class="{{ _css_class }}" role="search"{{ _extra_attributes|raw }}>
+  {% if textInput is defined %}
+    {% include '../ec-component-text-input/text-input.html.twig' with textInput %}
+  {% endif %}
+
+  {% if textInput is defined %}
```

Comment:
should be `button` instead of `textInput`

Reply:
Yes my mistake. Fixed in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7